mods/default/functions.lua

```diff
@@ -210,6 +210,61 @@ end
 
 
 --
+-- fence registration helper
+--
+function default.register_fence(name, def)
+	local fence_texture = "default_fence_overlay.png^" .. def.texture ..
+			"^default_fence_overlay.png^[makealpha:255,126,126"
+	def.drawtype = "fencelike"
+	-- allow almost everything to be overridden
```

Comment:
Comments should follow English capitalization rules -- which means starting with a capital letter.